### PR TITLE
Codefolding atom style

### PR DESCRIPTION
**Pull request by AnshulMalik**
Associated Issue: #2588 

Updated folders to use atom like style

### Screenshots

codefolding-atom

### CODE REVIEW

**Comment by jasonLaster** [comment]
@AnshulMalik this looks really good!

**Comment by jasonLaster** [comment]
looks like a simple linting question

```
src/components/Editor/Editor.css
 186:1  ✖  Expected no more than 1 empty line(s)   max-empty-lines
```

you can run `yarn lint-css` too

**Comment by jasonLaster** [comment]
How about we make code folding markers on top of breakpoints white? :)

File: src/components/Editor/Editor.css [hunk 1]
```diff
@@ -156,3 +156,45 @@ html[dir="rtl"] .editor-mount {
...
+
+.CodeMirror-foldgutter-folded::before {
+  border-left-color: var(--theme-body-color);
+  left: 4px;
+  top: 3px;
+}
+
+.new-breakpoint .CodeMirror-foldgutter-folded::after {
+  border-left-color: var(--theme-selection-background);
+}
```
Comment (jasonLaster) [inline src/components/Editor/Editor.css]: Lets move the css to the [source-editor][1] package.

[1]: https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-source-editor/src/source-editor.css#L180

**Comment by AnshulMalik** [comment]
@jasonLaster. I am also in favour of making it white.

**Comment by jasonLaster** [comment]
Looks great Anshul

**Comment by jasonLaster** [comment]
closing as the next PR will be in core

**Pull request closed without merging**